src/components/NcNoteCard/NcNoteCard.vue

```diff
@@ -139,20 +151,27 @@ export default {
 
 <style lang="scss" scoped>
 .notecard {
+	--note-card-padding: calc(2 * var(--default-grid-baseline));
 	color: var(--color-main-text) !important;
 	background-color: var(--note-background) !important;
-	border-inline-start: 4px solid var(--note-theme);
+	border-inline-start: var(--default-grid-baseline) solid var(--note-theme);
 	border-radius: var(--border-radius);
-	margin: 1rem 0;
-	margin-top: 1rem;
-	padding: 1rem;
+	margin: 1lh 0;
+	padding: var(--note-card-padding);
 	display: flex;
 	flex-direction: row;
-	gap: 1rem;
+	gap: var(--note-card-padding);
+
+	&__heading {
+		font-weight: 600;
+		font-size: 20px; // Same as icon
+	}
 
-	&__icon--heading {
-		margin-bottom: auto;
-		margin-top: 0.3rem;
+	&__icon {
+		&--heading {
+			// 20px heading font size * 1.5 line height = 30px. 30px - 20px icon size = 10px. 10px / 2 for alignment
+			margin-block: 5px auto;
```

Comment:
Might be overcomplicated, but
```suggestion
			font-size: 20px // or use variable
			margin-block: calc((1lh - 20px) / 2) auto;
```

Reply:
done